src/engine/strat_engine/backstore/metadata/static_header.rs

```diff
@@ -235,6 +235,33 @@ impl StaticHeader {
             }
         }
 
+        // Action taken when both sigblocks are interpreted as valid.
```

Comment:
This is not quite correct. It's more like: ```Action taken when both sigblock locations are analyzed without encountering an error.``` The important distinction is that if an error were encountered, then it would be uncertain whether their was Stratis metadatad there or not. In this method, it is completely known whether there is a sigblock there.